joblib/test/test_pool.py

```diff
@@ -406,7 +391,7 @@ def test_memmaping_on_dev_shm():
         # pickling procedure generate a .pkl and a .npy file:
         assert_equal(len(os.listdir(pool_temp_folder)), 2)
 
-        b = np.ones(100, dtype=np.float64)
+        b = np.ones(100, dtype=np.float64) * 2
```

Comment:
> otherwise the 2 arrays a and b have the same content and I cannot test that b
> has been memmaped to a new file.

Good point. You should add a comment about that.

Are we memmapping with 'c' or 'r'? The reason that I am asking is that if
we are not, we need to memmap two different arrays (is the sense of
identity) with the same content to different files.